### ISSUE TITLE
I can not connect to SIDN?

### ISSUE THREAD

**Issue by kvanleeuwen20** [metaregistrar/php-epp-client#169]
hello,

I've set my credentials and whitelisted my IPV4 addresses with SIDN..
But still I get the message that the login failed.... (I instantiate the service by creating a new sidnEppConnection(); ). Any clues?

**Comment by foxxlpaul** [comment]
Are you sure you are connecting with that whitelisted IP? Maybe you are connecting with IPv6 or another IPv4 address on the same machine?

**Comment by metaregistrar** [comment]
Please also check that your firewall is not blocking outgoing traffic for port 700. Nowadays most firewalls are standard blocking everything going out except the obvious ports.

**Comment by metaregistrar** [comment]
Hi Kevin,

Please use $conn = eppConnection::create('configfile',true) to enable debugging. Then, all commands will show a log of what happened, making it easier to see what goes wrong.

**Comment by metaregistrar** [comment]
yes, sidn needs that field. Please use sidnEppCreateDomainRequest instead of eppCreateDomainRequest to create a domain name with SIDN

**Comment by kvanleeuwen20** [comment]
Clear, thank you! It actually goes wrong when trying to create a new contact instead of registrating a new domain name. So here:

 try
{
        $contactinfo = new eppContact(new eppContactPostalInfo($name, $city, $country, $organization, $address, null, $postcode, Metaregistrar\EPP\eppContact::TYPE_LOC), $email, $telephone);

        $contact = new eppCreateContactRequest($contactinfo);
        if ($response = $conn->request($contact)) 
		{
            echo "Contact created on " . $response->getContactCreateDate() . " with id " . $response->getContactId() . "\n";
            return $response->getContactId();
        }
    }catch(eppException $e)
	{
        echo $e->getMessage() . "\n";
    }
    return null;

Do I need to use SidnEppContact?

**Comment by kvanleeuwen20** [comment]
Yes, I found it, it works now, thank you very much!

**Comment by kvanleeuwen20** [comment]
one more question: 
$transfer = new metaregEppTransferExtendedRequest(eppTransferRequest::OPERATION_REQUEST,$domain);
$response = $conn->request($transfer);

Does this also need to be sidnEppTransferExtendedRequest? If yes, the package loader can not find the class. And in the provided example the metaregEppTransferExtendedRequest is not found? What do I need to use to transfer domains?
